plugins/techdocs-node/src/stages/publish/azureBlobStorage.ts

```diff
@@ -78,6 +78,29 @@ export class AzureBlobStoragePublish implements PublisherBase {
       );
     }
 
+    const legacyPathCasing =
+      config.getOptionalBoolean(
+        'techdocs.legacyUseCaseSensitiveTripletPaths',
+      ) || false;
+
+    // Give more priority for connectionString, if configured, return the AzureBlobStoragePublish object here itself
+    const connectionStringKey =
+      'techdocs.publisher.azureBlobStorage.connectionString';
+    const connectionString = config.getOptionalString(connectionStringKey);
+    if (connectionString) {
+      logger.info(
+        `using ${connectionStringKey} string to create BlobServiceClient`,
```

Comment:
```suggestion
        `Using '${connectionStringKey}' configuration to create storage client`,
```

just to not show implementation details like the class name unnecessarily (risks getting lost in refactor anyway)